shopfloor/services/zone_picking.py

```diff
@@ -803,10 +835,18 @@ def _set_destination_location(self, move_line, quantity, confirmation, location)
             )
             return (location_changed, response)
         # destination location set to the scanned one
-        self._write_destination_on_lines(move_line, location)
+
         stock = self._actions_for("stock")
-        stock.mark_move_line_as_picked(move_line, quantity)
-        stock.validate_moves(move_line.move_id)
+        move_lines = move_line
+        if package:
+            quantity = 0
```

Comment:
```suggestion
            quantity = None
```
Sorry the quantity must be None instead of 0 otherwise noting will be set done